app/jobs/scheduled/poll_mailbox.rb

```diff
@@ -116,5 +130,39 @@ def mark_as_errored!
       $redis.zadd(POLL_MAILBOX_ERRORS_KEY, now, now.to_s)
     end
 
+    def poll_mailbox_timeout_key(date)
+      "#{POLL_MAILBOX_TIMEOUT_ERROR_KEY}-#{date}"
+    end
+
+    def poll_mailbox_timeout_count_key
+      "#{POLL_MAILBOX_TIMEOUT_ERROR_KEY}-count"
+    end
+
+    private
+
+    def notify_admin(message_template)
+      site_contact_user = Discourse.site_contact_user
+
+      if site_contact_user != Discourse.system_user
+        SystemMessage.create_from_system_user(site_contact_user, message_template)
+      elsif contact_email = SiteSetting.contact_email
+        client_message = RejectionMailer.send_rejection(
+          message_template, contact_email, {}
+        )
+
+        Email::Sender.new(client_message, message_template).send
+      end
```

Comment:
@SamSaffron  We will try PM but if no `site_contact_user` has been set, we email the `contact_email`. If both settings have not been set....the admin dashboard alert is still there.